MiddlesexMA/src/main/java/com/openlattice/middlesex/MiddlesexBookings1.java

```diff
@@ -95,102 +95,98 @@ public static void main( String[] args ) throws InterruptedException {
         Flight flight = Flight.newFlight()
                 .createEntities()
                 .addEntity( "suspect" )
-                .to( "MSOSuspects" )
-                .ofType( new FullQualifiedName( "general.person" ) )
-                .key( new FullQualifiedName( "nc.SubjectIdentification" ) )
-                .addProperty( new FullQualifiedName( "nc.PersonGivenName" ) )
-                .value( row -> row.getAs( "f_name" ) ).ok()
-                .addProperty( new FullQualifiedName( "nc.PersonMiddleName" ) )
-                .value( row -> row.getAs( "m_name" ) ).ok()
-                .addProperty( new FullQualifiedName( "nc.PersonSurName" ) )
-                .value( row -> row.getAs( "l_name" ) ).ok()
-                .addProperty( new FullQualifiedName( "nc.SSN" ) )
-                .value( row -> row.getAs( "ssno" ) )
-                .ok()
-                .addProperty( new FullQualifiedName( "nc.PersonRace" ) )
-                .value( row -> row.getAs( "race" ) ).ok()
-                .addProperty( new FullQualifiedName( "nc.MaritalStatus" ) )
-                .value( row -> row.getAs( "marit" ) ).ok()
-                .addProperty( new FullQualifiedName( "nc.PersonBirthDate" ) )
-                .value( MiddlesexBookings1::safeDOBParse )
-                .ok()
-                .addProperty( new FullQualifiedName( "nc.PersonBirthPlace" ) )
-                .value( row -> row.getAs( "birth" ) )
-                .ok()
-                .addProperty( new FullQualifiedName( "nc.SubjectIdentification" ) )
-                .value( MiddlesexBookings1::getSubjectIdentification ).ok()
-                .ok()
+                    .to( "MSOSuspects" )
```

Comment:
this also doesn't really matter, but moving forward it would be nice to use the less verbose format for directly reading values from rows where we don't do any string processing (i.e. `.addProperty( fqn, headerName )`) -- it's a bit easier to read